dataprofiler/profilers/profile_builder.py

```diff
@@ -1911,6 +1936,50 @@ def report(self, report_options: dict = None) -> dict:
 
         return _prepare_report(report, output_format, omit_keys)
 
+    @classmethod
+    def load_from_dict(
+        cls,
+        data,
+        options: dict | None = None,
+    ):
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+        :param options: options for loading column profiler params from dictionary
+        :type options: Dict | None
+
+        :return: Profiler with attributes populated.
+        :rtype: StructuredProfiler
+        """
+        chi2_matrix = data.pop("chi2_matrix")
+        correlation_matrix = data.pop("correlation_matrix")
+
+        structured_profiler = super().load_from_dict(data)
+
+        structured_profiler.times = defaultdict(float, structured_profiler.times)
```

Comment:
shouldn't need to do this if we fix the above.

Reply:
Fixed above